shared/js/media/remote_controls.js

```diff
@@ -160,21 +164,24 @@ MediaRemoteControls.prototype._setupBluetooth = function(callback) {
     self.defaultAdapter.ona2dpstatuschanged = a2dpConnectionHandler;
     self.defaultAdapter.onscostatuschanged = scoConnectionHandler;
 
-    if (callback)
+    if (callback) {
       callback();
+    }
   }
 
   function playstatusHandler() {
-    if (self._commandListeners['updateplaystatus'].length > 0)
+    if (self._commandListeners.updateplaystatus.length > 0) {
```

Comment:
Does jshint really complain about the use of string indexing here?  I agree that the code is better with your change here, but I didn't write this code. So if this change is not required for jshint compliance, please ask dkuo to review the changes to this file.